backend/service/audit/setup.go

```diff
@@ -156,9 +159,22 @@ func (c *client) poll(interval time.Duration) {
 		}
 	}
 
+	lockID := c.storage.GetLockID(auditEventSinkLockId)
 	ticker := time.NewTicker(interval)
```

Comment:
we should jitter the interval to minimize the chance that everything will be trying to take the lock at once